raiden/ui/cli.py

```diff
@@ -772,7 +772,7 @@ def print_step(description: str, error: bool = False) -> None:
             args["environment_type"] = environment_type
             args["extra_config"] = {"transport": {"matrix": {"available_servers": server_urls}}}
             args["one_to_n_contract_address"] = "0x" + "1" * 40
-            args["routing_mode"] = RoutingMode.PRIVATE
+            args["routing_mode"] = RoutingMode.LOCAL
```

Comment:
I think you changed the default here to testing purposes, should it be reverted?